tests/fixtures/phase3_fixtures.py

```
@@ -0,0 +1,562 @@
+        self._forced_failure = enabled
+
+    def _is_error_retryable(self, error):
+        """
+        Classifies all errors as retryable for circuit breaker logic.
+
+        Returns:
+            bool: Always returns True, indicating every error is considered retryable.
+        """
+        return True
+
+    def get_stats(self) -> Dict:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)